core/iwasm/aot/arch/aot_reloc_xtensa.c

```diff
@@ -40,6 +60,28 @@ static SymbolMap target_sym_map[] = {
 
     REG_SYM(__modsi3),
     REG_SYM(__divdi3),
+
+    REG_SYM(__udivdi3),
+    REG_SYM(__unorddf2),
+    REG_SYM(__adddf3),
+    REG_SYM(__eqdf2),
+    REG_SYM(__muldf3),
+    REG_SYM(__gedf2),
+    REG_SYM(__ledf2),
+    REG_SYM(__fixunsdfsi),
+    REG_SYM(__floatunsidf),
+    REG_SYM(__subdf3),
+    REG_SYM(__nedf2),
+    REG_SYM(__fixdfsi),
+    REG_SYM(__moddi3),
+    REG_SYM(__extendsfdf2),
+    REG_SYM(__truncdfsf2),
+    REG_SYM(__gtdf2),
+    REG_SYM(__umoddi3),
+    REG_SYM(__floatdidf),
+    REG_SYM(__divsf3),
+    REG_SYM(sqrt),
+    REG_SYM(sqrtf),
```

Comment:
it seems awkward to export sqrt and sqrtf this way.
is there xtensa specific reasons to do this?

Reply:
no specific reasons, just find it in test. let me check.

Reply:
i suspect your original module (before aot compilation) had these undefined symbols.